lib/ProductOpener/Display.pm

```diff
@@ -4020,6 +4020,15 @@ HTML
 						;
 				}
 
+				if ($packager_codes{$canon_tagid}{cc} eq 'ie') {
```

Comment:
I'm not sure how this Ireland packager code branch got mixed up in your PR. Any ideas?

Reply:
No idea, my branch was clean locally, until I pull now.